base/multimedia.jl

```diff
@@ -225,6 +225,18 @@ function displayable(m::MIME)
     return false
 end
 
+
+macro display(exs...)
+    blk = Expr(:block)
+    for ex in exs
+        push!(blk.args, :(print($(sprint(show_unquoted,ex)*" = "))))
+        push!(blk.args, :(display(begin value=$(esc(ex)) end)))
+        push!(blk.args, :(println()))
+    end
```

Comment:
Sorry, I'm having a hard time understanding the whole display machinery. I'm not sure I can implement something like this for any kind of display. For instance, is it possible to ask a generic display to display a new line (i.e. some kind of `println()`)?